drivers/ovsSwitch.go

```diff
@@ -399,7 +398,7 @@ func (sw *OvsSwitch) DeletePort(epOper *OvsOperEndpointState) error {
 	}
 
 	// Delete the Veth pairs if required
-	if useVethPair {
+	if useVethPair && !skipVethPair {
```

Comment:
Yes, using Veth pairs is a temporary workaround. We should go back to using OVS internal interfaces.
Using Veth pairs vs. OVS internal interfaces takes 15-20% performance hit. So, eventually we'd like to go back to OVS internal interfaces. But, there is a bug in OVS where renaming the OVS interface to "eth0" inside the container namespace messes up ovsdb. Plan is, once that bug is fixed(or we contribute a fix to OVS), we'll go back to using OVS internal interfaces.